src/dataflow-types/src/types.rs

```diff
@@ -65,7 +65,7 @@ pub enum TailResponse<T = mz_repr::Timestamp> {
     /// A batch of updates over a non-empty interval of time.
     Batch(TailBatch<T>),
     /// The TAIL dataflow was dropped before completing. Indicates the end.
-    Dropped,
+    DroppedAt(Antichain<T>),
```

Comment:
Add comment what the antichain represents here.

Reply:
Thanks! Fixed.